### ISSUE TITLE
SamplerCustomAdvanced issue

### ISSUE THREAD

**Issue by Dalasito** [logtd/ComfyUI-HunyuanLoom#18]
Hi, I'm getting this message when trying to execute Flowedit:
`shape '[1, 1, 22, 30, 16, 1, 2, 2]' is invalid for input of size 44160`

Any ideas?

```
# ComfyUI Error Report
## Error Details
- **Node ID:** 13
- **Node Type:** SamplerCustomAdvanced
- **Exception Type:** RuntimeError
- **Exception Message:** shape '[1, 1, 22, 30, 16, 1, 2, 2]' is invalid for input of size 44160
## Stack Trace
```
  File "C:\pinokio\api\comfy.git\app\execution.py", line 327, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\pinokio\api\comfy.git\app\execution.py", line 202, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\pinokio\api\comfy.git\app\execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\pinokio\api\comfy.git\app\execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))

  File "C:\pinokio\api\comfy.git\app\comfy_extras\nodes_custom_sampler.py", line 651, in sample
    samples = guider.sample(noise.generate_noise(latent), latent_image, sampler, sigmas, denoise_mask=noise_mask, callback=callback, disable_pbar=disable_pbar, seed=noise.seed)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 984, in sample
    output = executor.execute(noise, latent_image, sampler, sigmas, denoise_mask, callback, disable_pbar, seed)

  File "C:\pinokio\api\comfy.git\app\comfy\patcher_extension.py", line 110, in execute
    return self.original(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 952, in outer_sample
    output = self.inner_sample(noise, latent_image, device, sampler, sigmas, denoise_mask, callback, disable_pbar, seed)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 935, in inner_sample
    samples = executor.execute(self, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar)

  File "C:\pinokio\api\comfy.git\app\comfy\patcher_extension.py", line 110, in execute
    return self.original(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 714, in sample
    samples = self.sampler_function(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **self.extra_options)

  File "C:\pinokio\api\comfy.git\app\env\lib\site-packages\torch\utils\_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\custom_nodes\ComfyUI-HunyuanLoom\nodes\flowedit_nodes.py", line 93, in flowedit_sample
    vt_tgt = model(zt_tgt, sigma*s_in, **extra_args)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 379, in __call__
    out = self.inner_model(x, sigma, model_options=model_options, seed=seed)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 915, in __call__
    return self.predict_noise(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\custom_nodes\ComfyUI-HunyuanLoom\nodes\flowedit_nodes.py", line 23, in predict_noise
    return sampling_function(self.inner_model, x, timestep, negative, positive, cfg, model_options=model_options, seed=seed)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 359, in sampling_function
    out = calc_cond_batch(model, conds, x, timestep, model_options)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 195, in calc_cond_batch
    return executor.execute(model, conds, x_in, timestep, model_options)

  File "C:\pinokio\api\comfy.git\app\comfy\patcher_extension.py", line 110, in execute
    return self.original(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\comfy\samplers.py", line 308, in _calc_cond_batch
    output = model.apply_model(input_x, timestep_, **c).chunk(batch_chunks)

  File "C:\pinokio\api\comfy.git\app\comfy\model_base.py", line 131, in apply_model
    return comfy.patcher_extension.WrapperExecutor.new_class_executor(

  File "C:\pinokio\api\comfy.git\app\comfy\patcher_extension.py", line 110, in execute
    return self.original(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\comfy\model_base.py", line 162, in _apply_model
    model_output = self.diffusion_model(xc, t, context=context, control=control, transformer_options=transformer_options, **extra_conds).float()

  File "C:\pinokio\api\comfy.git\app\env\lib\site-packages\torch\nn\modules\module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\env\lib\site-packages\torch\nn\modules\module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)

  File "C:\pinokio\api\comfy.git\app\comfy\ldm\hunyuan_video\model.py", line 328, in forward
    out = self.forward_orig(x, img_ids, context, txt_ids, attention_mask, timestep, y, guidance, control, transformer_options)

  File "C:\pinokio\api\comfy.git\app\comfy\ldm\hunyuan_video\model.py", line 311, in forward_orig
    img = img.reshape([img.shape[0]] + shape + [self.out_channels] + self.patch_size)
```

**Comment by EthicalCell** [comment]
Same issue here.. works with wrapper though

**Comment by fusillustrator** [comment]
> Same issue here.. works with wrapper though

what are you syaing bro? canb you be clear , i opened wrapper and i have no idea what nodes to conenct to . pls tell

**Comment by king-ayanda** [comment]
I got the same error... but it worked when I changed the resolution (of the uploaded video) to 960x544. Might have something to do with resolutions that are divisible by 16. For example, when I try 640x360 it doesn't work. But when I do 640x320 it works. (640 is divisible by 16, and 320 is divisible by 16). But idk.
